virt/nodes/virt-triggering-vm-failover-resolving-failed-node.adoc

```diff
@@ -6,15 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-If a node fails and xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[machine health checks] are not deployed on your cluster, virtual machines (VMs) with `runStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object.
-
-[NOTE]
-====
-If you installed your cluster by using xref:../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[installer-provisioned infrastructure] and you properly configured machine health checks, the following events occur:
-
-* Failed nodes are automatically recycled.
-* Virtual machines with xref:../../virt/nodes/virt-node-maintenance.adoc#run-strategies[`runStrategy`] set to `Always` or `RerunOnFailure` are automatically scheduled on healthy nodes.
-====
+If a node fails and link:https://access.redhat.com/articles/7057929[node health checks] are not deployed on your cluster, virtual machines (VMs) with `runStrategy: Always` configured are not automatically relocated to healthy nodes. To trigger VM failover, you must manually delete the `Node` object.
```

Comment:
@sabrinajess When i read it, it tells me what happen when we don't have node health check and we would need to delet node object manually but should we add a line about how to do it manually?

Reply:
@geetikakay I'm not sure why the fix version has vfuture (I haven't seen that before!), but I believe we're working on it now because the description requests a fix for 4.16+. Will check with SME on how to trigger it manually.

Reply:
Can you please check with SME is it possible and should we advise such thing for doing any manual operation.